websocket/frontend/src/todolist/TodoItem.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export default function TodoItem(props) {
+  return (
+    <li>{props.item.task}</li>
```

Comment:
This entire component at this stage is completely redundant isn't it?

Reply:
yep. in the next PR there will be more code specifically for the todo item